libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -281,6 +281,9 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM
     void update_EKF1(void);
     void update_EKF2(void);
 
+    // update of AOA and SSA estimations
+    void update_AOA_SSA(void);
```

Comment:
I don't see any buildup of history or filtering, so why have an update function instead of just calculating the AOA and SSA on demand when the access function is called? It could be a single access function that returns both values.